testsuite/features/secondary/srv_check_dist_channel_mapping_page.feature

```diff
@@ -0,0 +1,133 @@
+    And I select "Test-Channel-Deb-AMD64" from "channel_label" dropdown
+    And I click on "Update Mapping"
+    Then I should see a "SUSE Linux Enterprise Server 15 SP 4 iSeries modified" link in the table
+    And I should see a "test-channel-deb-amd64" link in the table
+
+  Scenario: Cleanup: delete the map created for x68_64 suse clients
+    When I follow the left menu "Software > Distribution Channel Mapping"
+    Given I see a "SUSE Linux Enterprise Server 15 SP 4 modified" link in the table
+    And I see a "x86_64" architecture description in the table
+    When I follow "SUSE Linux Enterprise Server 15 SP 4 modified"
+    Then I should see a "Update Distribution Channel Map" text
+    And I should see a "Delete Distribution Channel" link
+    When I follow "Delete Distribution Channel Mapping"
+    Then I should see a "Delete Distribution Channel Map" text
+    When I click on "Delete Mapping"
+    Then I should not see a "SUSE Linux Enterprise Server 15 SP 4 modified" link
+    
+  Scenario: Cleanup: delete the map created for x68_64 ubuntu clients
+    When I follow the left menu "Software > Distribution Channel Mapping"
+    Given I see a "Ubuntu 22.04.01 LTS modified" link in the table
+    And I see a "x86_64" architecture description in the table
+    When I follow "Ubuntu 22.04.01 LTS modified"
+    Then I should see a "Update Distribution Channel Map" text
+    And I should see a "Delete Distribution Channel" link
+    When I follow "Delete Distribution Channel Mapping"
+    Then I should see a "Delete Distribution Channel Map" text
+    When I click on "Delete Mapping"
+    Then I should not see a "Ubuntu 22.04.01 LTS modified" link
+    
+  Scenario: Cleanup: delete the map created for i586 clients
+    When I follow the left menu "Software > Distribution Channel Mapping"
+    Given I see a "SUSE Linux Enterprise Server 15 SP 4 iSeries modified" link in the table
+    And I see a "x86_64" architecture description in the table
+    When I follow "SUSE Linux Enterprise Server 15 SP 4 iSeries modified"
+    Then I should see a "Update Distribution Channel Map" text
+    And I should see a "Delete Distribution Channel" link
+    When I follow "Delete Distribution Channel Mapping"
+    Then I should see a "Delete Distribution Channel Map" text
+    When I click on "Delete Mapping"
+    Then I should not see a "SUSE Linux Enterprise Server 15 SP 4 iSeries modified" link
+
+  Scenario: Sanity check whether the page is in its default state
+    When I follow the left menu "Software > Distribution Channel Mapping"
+    Then I should see a "Distribution Channel Mapping" text
+    And I should see a "Channel List" link in the left menu
+    And I should see a "Package Search" link in the left menu
+    And I should see a "Manage" link in the left menu
+    And I should see a "Distribution Channel Mapping" link in the left menu
+    And I should see a "Create Distribution Channel Mapping" link
```

Comment:
```suggestion
```
FMPOV this can be skipped since you already checked that in the very first scenario. Here you only want to see that no distribution channel mapping exists after you deleted all.